misk-hibernate/src/main/kotlin/misk/hibernate/DbUnsharded.kt

```diff
@@ -3,4 +3,4 @@ package misk.hibernate
 /**
  * Marker interface for persistent entities that have a bounded growth and do not require sharding.
  */
-interface DbUnsharded<T : DbEntity<T>> : DbEntity<T>
\ No newline at end of file
+interface DbUnsharded<T : DbUnsharded<T>> : DbEntity<T>
```

Comment:
This more closely mirrors how `DbRoot`, `DbSharded`, etc are defined.

Reply:
yes!